action.yml

```
@@ -6,7 +6,13 @@ inputs:
     required: true
   api-key:
     description: "Tailscale API key"
-    required: true
+    required: false
+  oauth-client-id:
+    description: "tailscale OAuth ID"
+    required: false
+  oauth-secret:
+    description: "tailscale OAuth Secret"
```

nit: capitalize proper noun
```suggestion
    description: "Tailscale OAuth ID"
    required: false
  oauth-secret:
    description: "Tailscale OAuth Secret"
```